### ISSUE TITLE
100 Classic Books resets back to the Nintendo and Genius Sonority logo

### ISSUE THREAD

**Issue by Sagevonsuper** [DS-Homebrew/nds-bootstrap#1586]
**Your console:** Nintendo DS Lite

**Launch Method:** Flashcart (2019 R4i SDHC GOLD Pro)

**Version used:** TWiLightMenu v25.11.0 (nds-bootstrap v0.72.1)

**Game tested/used/applicable:** 100 Classic Books (NTR-YBNE-USA)

**Save file:** [SAV_PATH.zip](https://github.com/DS-Homebrew/nds-bootstrap/files/11912479/SAV_PATH.zip)

**Expected Behavior:** If you select "Start Reading", you should be able to select a book or go through the tutorial if you open the game for the first time. or if you select "Book Guide", you should be able to pick either two options: "Quiz" or "Rankings"

**Actual Behavior:** After selecting "Start Reading" or "Book Guide", the game will reset back to the Nintendo and the Genius Sonority logo. 

### Steps to reproduce
1. Open 100 Classic Books
2. Select "Start Reading" or "Book Guide"

### nds-bootstrap.ini

```[NDS-BOOTSTRAP]
USE_ROM_REGION = 1
NDS_PATH = fat:/roms/nds/100 Classic Books.nds
SAV_PATH = fat:/roms/nds/saves/100 Classic Books.sav
RAM_DRIVE_PATH = sd:/null.img
GUI_LANGUAGE = en
LANGUAGE = -1
REGION = 1
DSI_MODE = 0
EXTENDED_MEMORY = 0
DONOR_SDK_VER = 0
PATCH_MPU_REGION = 0
PATCH_MPU_SIZE = 0
FORCE_SLEEP_PATCH = 0
AP_FIX_PATH = fat:/_nds/nds-bootstrap/apFix.ips
DEBUG = 0
LOGGING = 1
MACRO_MODE = 0
SLEEP_MODE = 1
SOUND_FREQ = 0
CONSOLE_MODEL = 0
HOTKEY = 284
```

### Logs:
[NDSBTSRP.LOG](https://github.com/DS-Homebrew/nds-bootstrap/files/11912517/NDSBTSRP.LOG)

### Other notes
This should be able to apply the AP-Fix properly on nds-bootstrap, but it doesn't seem like it when I play this game on a DS Lite.

### Will you be available for future issues and/or replies?
Yes.

**Comment by RocketRobz** [comment]
Certain AP-fixes require the DS Memory Expansion Pak to be inserted.

**Comment by Sagevonsuper** [comment]
I see. Thanks for letting me know.

**Comment by RocketRobz** [comment]
Fixed by f3a652f3
